server/chat.ts

```diff
@@ -1851,7 +1851,7 @@ export const Chat = new class {
 			if (!migrationVersion) continue;
 			if (migrationVersion > curVersion) {
 				migrationsToRun.push({ version: migrationVersion, file: migrationFile });
-				Monitor.adminlog(`Pushing to migrationsToRun: ${migrationVersion} at ${migrationFile} - mainModule ${process.mainModule === module} !process.send ${!process.send}`);
+				Monitor.adminlog(`Pushing to migrationsToRun: ${migrationVersion} at ${migrationFile} - require.main ${require.main === module} !process.send ${!process.send}`);
```

Comment:
```suggestion
				Monitor.adminlog(`Pushing to migrationsToRun: ${migrationVersion} at ${migrationFile} - main module ${require.main === module} child process ${Boolean(process.send)}`);
```

Reply:
I don't want to flip the `!process.send` unnecessarily. Otherwise I have no issue with the log message change